test/integration/test/captions.test.js

```diff
@@ -1,9 +1,33 @@
 const clientHelper = require('../src/client-helper');
 const configHelper = require('../src/config-helper');
+const JobStatus = require('../../../dist/src/models/JobStatus').JobStatus;
+const client = clientHelper.getClient(configHelper.getApiKey());
 const InstanceStateError = require('../../../src/models/RevAiApiError').InvalidStateError;
 
+beforeAll(async (done) => {
+    const jobList = await client.getListOfJobs();
+    var jobId;
+    if(jobList != undefined) {
```

Comment:
should be `!==` in javascript we generally want to use `===` and `!==` over `!=` and `==` because the triple ones also enforce type